src/Semantics.js

```diff
@@ -196,6 +196,41 @@ Semantics.prototype.checkActionDicts = function() {
   }
 };
 
+Semantics.prototype.toRecipe = function() {
+  var sb = new common.StringBuffer();
+  sb.append('(function() {\n  var grammarFn = ');
+  sb.append(this.grammar.toRecipe());
+  sb.append('  var grammar = grammarFn.call(this);\n');
+  sb.append('  var semantics = grammar.semantics()');
+
+  ['Operation', 'Attribute'].forEach(function(type) {
+    var semantics = this[type.toLowerCase() + 's'];
```

Comment:
This name seems not quite right, b/c this variable doesn't hold a semantics instance. It's a list of all operations (resp. attributes), right?